DeepSpeech.ipynb

```diff
@@ -963,7 +963,7 @@
     "        # Execute operations of tower i on device i\n",
     "        with tf.device(available_devices[i]):\n",
     "            # Create a scope for all operations of tower i\n",
-    "            with tf.name_scope('tower_%d' % i) as scope:\n",
+    "            with tf.variable_scope('tower_%d' % i) as scope:\n",
```

Comment:
@kentsommer I haven't tested this. But this code is modeled off of a multi-gpu example from TensorFlow.

In particular, the [`cifar10_multi_gpu_train.py`](https://github.com/tensorflow/tensorflow/blob/r0.12/tensorflow/models/image/cifar10/cifar10_multi_gpu_train.py) multi-gpu code which uses the `tf.name_scope()` construct[[1](https://github.com/tensorflow/tensorflow/blob/r0.12/tensorflow/models/image/cifar10/cifar10_multi_gpu_train.py#L167)]
```python
    for i in xrange(FLAGS.num_gpus):
      with tf.device('/gpu:%d' % i):
        with tf.name_scope('%s_%d' % (cifar10.TOWER_NAME, i)) as scope:
          # Calculate the loss for one tower of the CIFAR model. This function
          # constructs the entire CIFAR model but shares the variables across
          # all towers.
          loss = tower_loss(scope)
          ...
```
not the `tf.variable_scope()` construct. 

I'm curious as to why you changed it from `tf.name_scope()` to `tf.variable_scope()`?